website/content/docs/k8s/operations/server-certificate-rotation.mdx

```diff
@@ -0,0 +1,27 @@
+---
+layout: docs
+page_title: Server Certificate Rotation
+sidebar_title: Server Certificate Rotation
+description: Rotate Server Certificate on Kubernetes Cluster safely
+---
+
+# Rotating Server Certificates
+
+As of Consul Helm version `0.29.0`, if TLS is enabled, new TLS certificates for the Consul Server
+are issued every time the TLS-init job runs. These certificates are signed by the same CA and will
```

Comment:
I'm thinking that tls-init might be an internal implementation that folks are not familiar with
```suggestion
are issued every time the Helm chart is upgraded. These certificates are signed by the same CA and will
```